### PR TITLE
Adding TFI (Friday Institute) back into the dropdown menu

### PR DESCRIPTION
**Pull request by hannahbergam**
This update is compiled by studying these two PRs from last summer to add/remove this same item:

Add TFI option PR: https://github.com/code-dot-org/code-dot-org/pull/35120

Remove TFI option PR: https://github.com/code-dot-org/code-dot-org/pull/36269

Ticket here: https://codedotorg.atlassian.net/browse/PLC-1157

### CODE REVIEW

**Comment by bencodeorg** [comment]
Based on # of lines changed, this looks _almost_ a revert of the most recent PR, but not exactly...what is the difference? I see a comment tense changed, anything else?

**Comment by hannahbergam** [comment]
> Based on # of lines changed, this looks _almost_ a revert of the most recent PR, but not exactly...what is the difference? I see a comment tense changed, anything else?

Some of the updates were in slightly different places, and I think part of the testing file didn't have to change. I am happy to do a pure revert instead- just was looking around the files as I was updating so ended up doing it old-school!

File: dashboard/app/models/pd/workshop.rb [hunk 1]
```diff
@@ -62,7 +62,7 @@ class Pd::Workshop < ApplicationRecord
     # organization.
     # Only current allowed values are "friday_institute" and nil.
     # "friday_institute" represents The Friday Institute,
-    # a regional partner whose model of virtual workshop was used
+    # a regional partner whose model of virtual workshop is being used
     # by several partners during summer 2020.
```
Comment (bencodeorg) [inline dashboard/app/models/pd/workshop.rb]: Nit: maybe update to include 2021? Or just remove any year reference, maybe virtual is forever at this point :).

**Review by bencodeorg — APPROVED** [review]
LGTM!

**Comment by bencodeorg** [comment]
Could you update the description and/or title to include something with the full name for TFI? I think the title might be a little misleading for future folks (virtual workshops are already an option)

File: dashboard/app/models/pd/workshop.rb (hunk 1 above)
Comment (hannahbergam) [reply dashboard/app/models/pd/workshop.rb]: 😂great point- will do!

**Comment by hannahbergam** [comment]
> Could you update the description and/or title to include something with the full name for TFI? I think the title might be a little misleading for future folks (virtual workshops are already an option)

Ahh yep good point! Thanks!

**Pull request merged**